contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+
+### Hog
+
+<BuiltBy people={["Marius Andra", "Anirudh Pillai"]} />
+
+What’s cooler than having your own query language? Having your own general purpose programming language, of course. That’s literally what we built: PostHog’s first ever programming language, Hog. 
+
+Things moved fast the week after Mykonos: we split up the existing product analytics team and built a new team to productize Hog. We plan to use Hog to build our CDP and messaging products, and might even pivot the entire company around it. It was just another PostHog hackathon, no big deal. Stay tuned for the aftermath.
+
+### RealTimeHog 3000
+
+<BuiltBy people={["James Greenhill", "Zach Waterfield", "Michael Matloka"]} />
+
+Seeing people using your product live boosts dopamine levels. Probably. PostHog does that – but right now we keep you waiting a bit. Before a PostHog event is available for querying, it goes through our ingestion pipeline, where users are identified and the events themselves enriched. The process takes seconds, rarely a minute – a delay imperceptible in analytical queries, but a dopamine decrease in the live view.
+
+Introducing RealTimeHog 3000, a livestream service powering our first truly real-time view of user activity. The process is simple: 1. Consume the raw events from the same Kafka topic as our ingestion service. 2. Stream them to you ASAP using server-sent events. The lack of person data is a fair trade-off, because it doesn’t get more live than this. User activity appears within milliseconds of happening on the other side of the world.
+
+Speaking of data being processed live, perhaps at some point you’ve wondered if PostHog actually scales, or just talks the talk. To dispel any doubts, we developed one extra feature: an anonymized stream of all events being captured globally, with only the geolocation included. Hundreds of thousands of events per minute, and they look great on a 3D globe on PostHog’s website, where each event is an arc from the user’s location to that of the relevant data center. Global scale, visualized for your pleasure.
+
+### SupportHog
+
+<BuiltBy people={["Dylan Martin", "Tiina Turban", "Marcus Hof", "Neil Kakkar"]} />
+
+Our support flow currently uses Zendesk and that goes through email. This causes three problems: emails sometimes bounce, there are long delays in checking emails, and it’s really clunky to add other team members via Cc. Simply put, it’s not the most optimal flow. Imagine if instead users could instead view and respond to their open support tickets without ever having to leave PostHog – well, that's exactly what we built.
+
+But that's not all, if you're using Zendesk then you could, in the future, add this view to your customer-facing website with just a few clicks.
```

Comment:
What's the plan with this project now?

Reply:
it's "just a few steps" as due to security there is backend side too. cc @neilkakkar  who's the TL for feature-success, who owns this. Maybe it could be a website roadmap thing people can vote on?

Reply:
Yeah, wouldn't actively work on this yet, will put it on our roadmap, good idea; eventually we'll either find the capacity to take this on (when current projects are done), or maybe new hires will take this on! Until then, on the back-burner.

Reply:
so the update here can say - if that's something you're interested like and follow the roadmap item.